extract_endpoint/src/crypt_utils.py

```diff
@@ -0,0 +1,13 @@
+import os
+import hmac
+import base64
+
+
+def get_salt(num_bytes=64) -> str:
```

Comment:
Is this function still needed?

Reply:
huh, that was just to convert the random bytes to a string to pass to the microservice, but we can use `secrets.token_hex()` instead of `secrets.token_bytes()` and get a string directly. 🔥